tests/commands/dev/test_create_environment.py

```diff
@@ -0,0 +1,66 @@
+import shutil
+import subprocess
+import sys
+
+import pytest
+
+from briefcase.exceptions import BriefcaseCommandError
+
+
+def test_create_environment(dev_command, first_app):
+    """Test that it creates a venv for the app."""
+    venv_path = (
+        dev_command.base_path / ".briefcase" / f"dev-web-venv-{first_app.app_name}"
+    )
+
+    # Ensure the environment path does not exist before the test so it can be created
+    if venv_path.exists():
+        shutil.rmtree(venv_path)
```

Comment:
This is an indicator of a problem - broadly speaking, test logic should *never* have conditional statements. 

If it's required, it's an indication that your test environment isn't isolated. You should be able to make concrete guarantees about what is (or isn't) in your test environment.

That said - in this specific case, it does overlap with one edge case that *should* be tested - what happens if the `.briefcase` directory *exists*, but the venv folder *doesn't*.